health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/components/pageComponents/SearchUserToReport.js

```
@@ -0,0 +1,227 @@
+
+
+    };
+
+    // -------- Render --------
+    return (
+        <PopUp
+            style={{ width: "800px", minHeight: "430px" }}
+            onClose={onClose}
+            heading={t("HCM_AM_SEARCH_USER")}
+            onOverlayClick={onClose}
+            children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)